sassdocs_helpers.rb

```diff
@@ -0,0 +1,110 @@
+require 'json'
+
+module SassdocsHelpers
+    def format_sassdoc_data(data)
+        raise "No data in data/sassdocs.json, run `npm install` to generate." unless data.respond_to?(:sassdoc)
+        data.sassdoc
+            .select { |doc|
+                # Remove private API entries
+                doc.access == 'public'
+            }
+            .select { |doc|
+                # Remove vendored files, for example SassMQ
+                !doc.file.path.start_with?('vendor')
+            }
+            .group_by {
+                # Group the docs by their 'group', for example 'Settings' or 'Helpers'
+                |doc| doc.group.first
```

Comment:
Minor, but there's inconsistency with the placement of the arguments here – in the two `select` calls the arguments are on the same line as the opening brace.